app/models/concerns/schema_validations.rb

```diff
@@ -68,6 +70,8 @@ def validate_presence(column, name, datatype)
         validate_logged :validates_inclusion_of, name, in: [true, false], message: :blank
       elsif !column.default.nil? && column.default.blank?
         validate_logged :validates_with, NotNilValidator, attributes: [name]
+      elsif datatype == :array
+        validate_logged :validates, name, presence: true, if: -> { public_send(name).nil? }
```

Comment:
kinda funky but we cant use exclusion here b/c of a change in rails 6.1 https://github.com/rails/rails/issues/41051, and this works fine